distance_tolerance: 0.02 # m, default: 0.01
angle_tolerance: 0.02 # rad, default: 0.01
linear_velocity: 0.1 # m/s, default: 0.5
angular_velocity: 12.56 # rad/s, default: 12.56 
loop_frequency: 20 # Hz, default: 20
cell_size: 0.18 # m, default 0.18
angular_gain: 0.8 #dimensionless 
linear_gain: 0.1 #dimensionless
row_init_position: 1
column_init_position: 1
